src/feature/existing-check.ts

```diff
@@ -0,0 +1,129 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import { Issue, IssuesOpenedEvent } from '@octokit/webhooks-definitions/schema';
+import axios from 'axios';
+
+import { URL_REGEX, EXCLUSION_LIST, urlsFromIssueBody } from './dupe-check';
```

Comment:
Could pull that out into another file, seems a bit awkward that it's shared but still in a specific feature file.